rfcs/0130-stalled-rfcs.md

```diff
@@ -0,0 +1,145 @@
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+## More Research Required
+
+If an RFC is opened and discussion reveals that more research is required, the author may now return their RFC to draft status to indicate that they are working on an update.
+
+This is already frequently done today, but this RFC codifies this process. The alternative, which is sometimes seen is that the NixOS RFC Steering Committee keeps pinging the RFC for months trying to move the RFC forward which is annoying and unproductive.
+
+## Can't Find Shepherds
+
+If an RFC is opened but a team of shepherds can't be assembled the RFC will now be closed.
+
+Currently there is no process here and the NixOS RFC Steering Committee is unsure what to do and tends to annoyingly ping these RFCs with no positive outcome for a few months then improvise. This RFC proposes a well-defined process to ensure that everyone understands the process.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+## Closing a PR may be demotivating.
+
+Struggling to find shepherds for an RFC may already be demotivating and having the RFC closed can be an additional flag that the RFC is struggling. This may be demoralizing to the RFC author.
+
+This issue is mitigated by having clear and well-thought-out messages to make it clear what is occurring at this stage of the process. This is hopefully better overall than the previously poorly defined state of forever waiting for more shepherds to appear.
+
+## Closing a PR may reduce visibility for the RFC.
+
+Leaving PRs open may increase the chance that a potential shepherd finds the RFC. Closing RFCs that can't find shepherds may worsen the thing that ails them.
+
+It is the author's opinion that having dozens of open RFCs will also reduce visibility from the ones that do have more interest and are more likely to find a full shepherd team. Therefore this is considered an acceptable cost. The addition of the `status: insufficient interest` label may also help to bring attention to these RFCs that could benefit from would-be shepherds.
+
+# Alternatives
+[alternatives]: #alternatives
+
+## Do Nothing
+
+Doing nothing leaves the behaviour to be taken when an RFC can not find a shepherd team undefined. This leads to inconsistent handling by the NixOS RFC Steering Committee and leaves community members unable to understand what to expect.
+
+## Leave the PR open and add a label
+
+This can reduce the burden from the NixOS RFC Steering Committee however it still clutters the list of "Open" RFCs with these RFCs that are not seeing forward progress. Ultimately it is a decision of definition of what "Open" means and this RFC takes the stance that if an RFC is stalled for too long it makes sense to remove it from the default search for "Open" RFCs.
+
```

Comment:
```suggestion

Due to similarity with [RFC 51 - Mark stale nixpkgs issues)[https://github.com/NixOS/rfcs/pull/51] which had a lot of backlash, extra effort was made to ensure that this was not too controversial. Notably a form post at https://discourse.nixos.org/t/stalled-rfcs-proposing-rfcs-with-no-interest/21952. It appears that to the Nix community this is in fact a substantially different case so closing is acceptable. 

```

Reply:
"forum post" typo. I think the main reason this is different from 51 is that it will be people exercising their judgement while manually closing them.

Reply:
I would probably stress not the backlash but «settled on labelling without closing after a lot of discussion» to point to the relevant part.

Otherwise I agree.

Reply:
Thanks for the suggestions. I've incorporated them all. Please double check.